modules/compute/src/main/java/org/apache/ignite/internal/compute/ComputeMessageTypes.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Message types for the Compute module.
  */
-@MessageGroup(groupName = "ComputeMessages", groupType = 6)
+@MessageGroup(groupType = 6, groupName = "ComputeMessages")
```

Comment:
What is the purpose of this change? :)

Reply:
groupType is first everywhere else, just a little style change